### PR TITLE
Group14

### PR DESCRIPTION
**Pull request by Dawoud-Husain**
This is groups 14 ShearInY pull request

### CODE REVIEW

File: ShearInY/ShearInYTests.c [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+//   if ( tempShape.shearing[1] == 0) {
+//     printf ("Error: The input value of the y shear is empty.\n");
+//   }
+//   else {
+//     return;
+//   }
+// } 
+
+// //dirty test 2: The input is out of the range of the matrix/vector
```
Comment (andrewlinington) [inline ShearInY/ShearInYTests.c:43]: Since these are commented out ill skip the -t tests

**Review by andrewlinington — APPROVED** [review]
Perfect looks good to merge

**Review by andrewlinington — APPROVED** [review]
looks good just needs to have its conflicts resolved

**Review by aidkvndfj — CHANGES_REQUESTED** [review]
image
I seem to be getting valgrind errors when running with and output.txt as the file. Please fix.

**Review by aidkvndfj — APPROVED** [review]

**Pull request merged**